docs/user-guide/configuration/index.md

```diff
@@ -70,6 +71,10 @@ You can access information about properties by hovering over the property name.
             <h4>Requires</h4>
             <p>A single job name or array of jobs that will trigger the job to run. Jobs defined with "requires: ~pr" are started by pull-request events. Jobs defined with "requires: ~commit" are started by push events. Jobs defined with "requires: ~sd@123:main" are started by job "main" from pipeline "123". Jobs defined with "requires: [deploy-west, deploy-east] are started after "deploy-west" and "deploy-east" are both done running successfully. "Note: ~ jobs denote an OR functionality, jobs without a ~ denote join functionality.</p>
         </div>
+        <div id="sourcePaths" class="hidden">
+            <h4>Source Paths</h4>
+            <p>You can optionally specify source paths that will trigger a job upon modification. In this example, the "main" job will only run if changes are made to things under the "src/app/" directory or the "screwdriver.yaml" file. Due to limitations in Git APIs, this feature is only available for Github SCM.</p>
```

Comment:
`limitations in Bitbucket and Gitlab APIs...`
(not sure if it's a limitation in Gitlab though since we didn't attempt Gitlab, only Bitbucket.
That, or we can just say it's only available for Github SCM without any explanation.

Reply:
👍 Updated